.circleci/config.yml

```diff
@@ -108,7 +108,7 @@ jobs:
           name: Integration Tests
           no_output_timeout: 40m
           command: |
-            set -m
+            set -em
```

Comment:
What does this change do?

Reply:
makes sure errors are not being suppressed: https://support.circleci.com/hc/en-us/articles/115015733328-Tests-fail-but-job-finishes-successfully